src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -125,11 +128,12 @@ public function Link($action = null)
      */
     public function view($request)
     {
-        if (!$this->record->canView()) {
+        // Assume item can be viewed if canView() isn't implemented
+        if ($this->record->hasMethod('canView') && !$this->record->canView()) {
```

Comment:
Consistency with what?

Reply:
I think that would be a breaking change, since currently someone _could_ have data being used in this way which _doesn't_ implement a `canView()` check. Maybe open a card to enforce this in a major release?

Reply:
I think we need to document this well. But otherwise we can live with this.